posix.mak

```diff
@@ -585,7 +585,7 @@ $G/%-${LATEST} :
 
 # Clone all main D repositories
 ${DMD_DIR} ${DRUNTIME_DIR} ${PHOBOS_DIR} ${TOOLS_DIR} ${INSTALLER_DIR} ${DUB_DIR}:
-	git clone ${GIT_HOME}/$(notdir $(@F)) $@
+	git clone --depth=1 ${GIT_HOME}/$(notdir $(@F)) $@
```

Comment:
It takes quite a long time to do a full checkout of `dmd` and it isn't really required.

Reply:
Git history is needed for `git describe` to work properly, so this might not be a good idea.